corehq/apps/app_manager/views/releases.py

```diff
@@ -290,11 +291,10 @@ def save_copy(request, domain, app_id):
             timer = datadog_bucket_timer('commcare.app_build.new_release', tags=[],
                                          timing_buckets=(1, 10, 30, 60, 120, 240))
             with timer:
-                copy = app.make_build(
-                    comment=comment,
-                    user_id=user_id,
-                )
-                copy.save(increment_version=False)
+                result = make_app_build(app, comment, user_id)
+                if isinstance(result, HttpResponse):
```

Comment:
see my note above, but I think this should instead be a try/except block instead of checking the return type.

Reply:
Thanks @proteusvacuum , updated.